GameData/RealSolarSystem/Compatibility/CustomAsteroids.cfg

```diff
@@ -707,19 +707,19 @@ AsteroidSets
 
 		asteroidTypes
 		{
-			key = 0.8 CaAsteroidIcy
+			key = 0.98 CaAsteroidIcy
 			// key = 0.1 CaCometActive
```

Comment:
Please also remove references to `CaCometActive`. It really shouldn't be used in new configs.